bq/insert.go

```diff
@@ -57,6 +60,16 @@ func (in *BQInserter) InsertRows(data interface{}) error {
 	}
 }
 
+// Caller should check error, and take appropriate action before calling again.
+func (in *BQInserter) InsertRows(data []interface{}) error {
+	in.rows = append(in.rows, data...)
```

Comment:
Do we care that this could completely blow past the line limit in a big way if `data` has a lot of elements?